### ISSUE TITLE
UI inconsistency: article/web button placement (and possible room for second favorite action in upper bar)

### ISSUE THREAD

**Issue by alainsr** [arunkumar9t2/lynket-browser#137]
I like being able to quickly switch from article view to web view, but I noticed that when I'm in article view the button for switching to web version is up there, left of the (single) favorite action button. Then, in web view, the bottom bar has not 4 buttons anymore but 5. Meanwhile, there's still only one favorite action button in the upper bar.

Suggestion: keep the 5 button bottom bar everywhere and allow 2 favorite buttons.

**Comment by arunkumar9t2** [comment]
Agreed, let me investigate how this could be done. I need to investigate this as I too thought of the same thing but decided against it. Can't remember why. I will check.
